### PR TITLE
Numbers operations

### PR DESCRIPTION
**Pull request by saurabh077**
check for the numbers

### CODE REVIEW

File: Numbers.c [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include<stdio.h>
+#define MAX_STRING_SIZE 15
```
Comment (riteshja88) [inline Numbers.c]:
```
#define MAX_STRING_LEN 14
```

File: Numbers.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+int main(){
+	char numbers[][MAX_STRING_SIZE] = {"one","two", "three","four","five","six", "seven","eight","nine", "ten"};
```
Comment (riteshja88) [inline Numbers.c]:
```
-char numbers[][MAX_STRING_SIZE]
+char numbers[][MAX_STRING_LEN + 1]
```

File: Numbers.c [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+	int i=0;
+	for(i=0;i<10;i++)
```
Comment (riteshja88) [inline Numbers.c]: please use curly braces for for loop

Comment (saurabh077) [reply Numbers.c]: yes. Did the suggested changes.

File: Numbers.c [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
+#define MAX_STRING_LEN 14
+int main(){
```
Comment (riteshja88) [inline Numbers.c]: can we use curly brace on next line for functions?

File: Numbers.c (hunk 3 above)
Comment (riteshja88) [reply Numbers.c]: can we use curly brace on same line for non-functions?

File: Numbers.c (hunk 4 above)
Comment (saurabh077) [reply Numbers.c]: Yes.

**Review by riteshja88 — APPROVED** [review]

File: Numbers.c [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
+#define MAX_STRING_LEN 14
+int main()
+{
+	char numbers[][MAX_STRING_LEN+1] = {"one","two", "three","four","five","six", "seven","eight","nine", "ten"};
+	int i=0;
+	for(i=0;i<10;i++){
+		printf("\n%d : %s", i+1,numbers[i]);
```
Comment (riteshja88) [inline Numbers.c]: `\n` should be at then end of `printf`

**Review by riteshja88 — APPROVED** [review]

File: Numbers.c (hunk 5 above)
Comment (saurabh077) [reply Numbers.c]: Done

**Comment by saurabh077** [comment]
Should I merge then?

**Pull request merged**